pom.xml

```diff
@@ -174,6 +174,7 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
             <version>${spring.version}</version>
+            <optional>true</optional>
```

Comment:
I would just add a comment here, with use-cases, when you need this dep and when not.